utils/b2d.go

```diff
@@ -145,6 +145,10 @@ func (b *B2dUtils) DownloadLatestBoot2Docker() error {
 		return err
 	}
 
+	return b.DownloadBoot2Docker(latestReleaseUrl)
+}
+
+func (b *B2dUtils) DownloadBoot2Docker(latestReleaseUrl string) error {
```

Comment:
So this name is kind of a misnomer in this PR, huh?  Since it looks like it gets used to get RancherOS ISO too.